spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/db/container/DataflowCluster.java

```diff
@@ -396,32 +414,61 @@ private SkipperContainer<?> buildSkipperContainer(
 		}
 
 		if (oauthContainer != null) {
```

Comment:
Why were we checking if `oauthContainer != null` anyways? We should assert it is not null way above otherwise there is something else seriously wrong w/ this test.

Reply:
The DataflowCluster is used to execute database tests where there are no oauthContainer